styles.css

```diff
@@ -127,3 +132,172 @@ body {
   opacity: 1;
   width: 100%;
 }
+
+@media (min-width: 1900px) {
+  .wrapper {
+    width: 85%;
+  }
+
+  .logo {
+    padding: 40px 0 40px 97px;
+  }
+
+  .logo__circle_color_blue,
+  .logo__circle_color_dark-blue {
+    margin-left: 53px;
+  }
+
+  .menu {
+    padding-top: 0;
+    padding-right: 39px;
+  }
+
+  .menu__item {
+    font: 500 18px/21px Roboto, arial, helvetica, sans-serif;
+  }
+
+  .header {
+    margin-bottom: 10%;
+  }
+
+  .main {
+    padding: 38px 50% 0 50px;
+    background: top -30px right 6% /47% no-repeat url(img/img.svg);
+  }
+
+  .main__title {
+    font: bold 66px/88px 'pt_serif', arial, helvetica, sans-serif;
+    margin-bottom: 35px;
+  }
+
+  .main__description {
+    max-width: 60%;
+    font: 22px/26px Roboto, arial, helvetica, sans-serif;
+  }
+
+  .main__button {
+    font: 500 18px/21px Roboto, arial, helvetica, sans-serif;
+    padding: 17px 28px;
+    margin-left: 2px;
+    margin-bottom: 25%;
+  }
+}
+
+@media (max-width: 768px) {
+  #checkbox {
+    opacity: 0;
+  }
+
+  .burger__menu {
+    display: flex;
+    width: 25px;
+    padding-top: 4px;
+    border-bottom: 3px solid #0056ec;
+  }
+
+  .burger {
+    display: block;
+    position: relative;
+    margin: 27px 8px 0 0;
+    z-index: 20;
+  }
+
+  .burger::after, .burger::before {
+    display: flex;
+    position: relative;
+    content: "";
+    border-bottom: 3px solid #0056ec;
+    padding-bottom: 4px;
+  }
+
+  #checkbox:checked ~ label .burger::before {
+    position: relative;
+    transform: rotate(45deg);
+    top: 7px;
+  }
+
+  #checkbox:checked ~ label .burger::after {
+    position: relative;
+    transform: rotate(-45deg);
+    top: -4px;
+    right: 3px;
+  }
+
+  #checkbox:checked ~ label .burger__menu {
+    border: none;
+  }
+
+  .menu {
+    display: none;
+  }
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/FirstWebsait/pullRequest?prid=6278784)